dojo/unittests/test_github_vulnerability.py

```diff
@@ -0,0 +1,37 @@
+from django.test import TestCase
+from dojo.models import Test
+from dojo.tools.github_vulnerability.parser import GithubVulnerabilityParser
+
+
+class TestGithubVulnerabilityParser(TestCase):
+    def test_parse_without_file_has_no_findings(self):
+        parser = GithubVulnerabilityParser(None, Test())
+        self.assertEqual(0, len(parser.items))
+
+    # sample with zero vulnerability
+    def test_parse_file_with_no_vuln_has_no_findings(self):
+        testfile = open("dojo/unittests/scans/github_vulnerability/github-0-vuln.json")
+        parser = GithubVulnerabilityParser(testfile, Test())
+        self.assertEqual(0, len(parser.items))
+
+    # sample with one vulnerability
+    def test_parse_file_with_one_vuln_has_one_findings(self):
+        testfile = open("dojo/unittests/scans/github_vulnerability/github-1-vuln.json")
+        parser = GithubVulnerabilityParser(testfile, Test())
+        self.assertEqual(1, len(parser.items))
+
+        findings = parser.items[0]
+        self.assertEqual(findings.title, "aabbccddeeff1122334401")
+        self.assertEqual(
+            findings.description,
+            "This is a sample description for sample description from Github API.",
+        )
+        self.assertEqual(
+            findings.severity, "Critical"
+        )
```

Comment:
This fails in our Travis checks
![image](https://user-images.githubusercontent.com/46459665/85314347-0242cd00-b47f-11ea-9048-bb87ad105592.png)

Reply:
fixed